lib/ruleHelper.js

```diff
@@ -258,8 +315,15 @@ RuleHelper.prototype = {
             }
             ruleCheck.properties.forEach((propertyId) => {
                 const argument = node.arguments[propertyId];
+                const details = {};
                 if (this.shouldCheckMethodCall(node, ruleCheck.objectMatches)
-                    && !this.allowedExpression(argument, ruleCheck.escape)) {
+                    && !this.allowedExpression(argument, ruleCheck.escape, details)) {
+                    // Include the additional details if available (e.g. name of a disallowed variable
+                    // and the position of the expression that made it disallowed).
+                    if (details.message) {
+                        this.context.report(node, `Unsafe call to ${this.getCodeName(node.callee)} for argument ${propertyId} (${details.message})`);
+                        return;
+                    }
```

Comment:
@mozfreddyb my apologies, I totally forgot to mention that my diff didn't include any test case that does cover the linting message reported here with the additional details, it would be good to add one in `tests/rules/method.js`.